test/baseline/file/FileUpload.test.js

```diff
@@ -0,0 +1,413 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-env mocha */
+
+// ****************************************************************
+//  THIS FILE IS USED TO GENERATE TEST DATA FOR THE files-seeder.js
+//    SEQUELIZE SEEDER.
+//  IT IS NOT A LEGITIMATE TEST FILE
+// ****************************************************************
+//  DO NOT RUN THIS FILE UNLESS YOU WANT TO GENERATE NEW TEST DATA
+// ****************************************************************
+
+const { request, expect, testArtistUserId, testAccessToken } = require('../../testConfig')
+const MockAccessToken = require('../../MockAccessToken')
+
+const audioSourceFilePath = '/var/www/api/test/media/audio/'
+
+let response = null
+let trackId = null
+let fileName = null
+
+describe.skip('baseline / file upload endpoint tests', () => {
+  MockAccessToken(testArtistUserId)
+
+  it('should reject a PUT request without access token', async () => {
+    response = await request.put('/user/tracks/:id/file')
+
+    expect(response.status).to.eql(401)
+  })
+  it('should reject a PUT request with with a valid access token, but no track id ', async () => {
+    response = await request.put('/user/tracks/:id/file')
+      .set('Authorization', `Bearer ${testAccessToken}`)
+
+    expect(response.status).to.eql(500)
+  })
+  it('should reject a PUT request with with a valid access token, but invalid track id ', async () => {
+    response = await request.put('/user/tracks/1234-asdf/file')
+      .set('Authorization', `Bearer ${testAccessToken}`)
+
+    expect(response.status).to.eql(500)
+  })
+})
+
+describe.skip('ALBUM: Best Album Ever', () => {
+  MockAccessToken(testArtistUserId)
+
+  // Ergonomic interactive concept,  Laurie Yost
+  it('should process audio file / track 01', async () => {
+    trackId = '44a28752-1101-4e0d-8c40-2c36dc82d035'
+    fileName = 'whiteNoise5s1.m4a'
+
+    response = await request.put(`/user/tracks/${trackId}/file`)
+      .set('Authorization', `Bearer ${testAccessToken}`)
+      .attach('files', `${audioSourceFilePath}${fileName}`)
+
+    expect(response.status).to.eql(200)
+  })
+
+  // Customer-focused fresh-thinking groupware Miss Lewis Ondricka
+  it('should process audio file / track 02', async () => {
+    trackId = '706cff12-ba44-49f7-8982-98b3996a2919'
+    fileName = 'whiteNoise5s2.m4a'
+
+    response = await request.put(`/user/tracks/${trackId}/file`)
+      .set('Authorization', `Bearer ${testAccessToken}`)
+      .attach('files', `${audioSourceFilePath}${fileName}`)
+
+    expect(response.status).to.eql(200)
+  })
+
+  // Configurable bandwidth-monitored definition Roberto Romaguera
+  it('should process audio file / track 03', async () => {
```

Comment:
What are the different things these tests test? It kind of looks like the same test?